metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/MediaProcessPluginMetadata.java

```diff
@@ -9,7 +9,7 @@
 public class MediaProcessPluginMetadata extends AbstractExecutablePluginMetadata {
 
   private static final ExecutablePluginType pluginType = ExecutablePluginType.MEDIA_PROCESS;
-  private ThrottlingLevelValuePair.ThrottlingLevel throttlingLevel;
+  private String throttlingLevel;
```

Comment:
I think you should bring the enum back... see my other comment.

Reply:
It's back!